specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2018-09-30-preview/openShiftManagedClusters.json

```diff
@@ -731,6 +731,10 @@
             "tenantId": {
               "type": "string",
               "description": "The tenantId associated with the provider."
+            },
+            "customerAdminGroupId": {
```

Comment:
>customerAdminGroupId [](start = 13, length = 20)

How will this impact existing clients. What will the behavior be when it's missing? Any back-compat concerns?

Reply:
Creates with the field missing will just not use/get the new feature.  Clients should be able to handle receiving an unexpected field and just ignore the new field on gets. If they do an update call and the field is missing we will just not update the field and it won't undo work done with a newer client.

Reply:
In simpler terms. This is an optional field. If it is missing old behavior is unchanged. If we add the field it will get the new functionality. With this being a preview version this should be fine. CC: @amanohar

Reply:
Makes sense.

---
In reply to: [270517299](https://github.com/Azure/azure-rest-api-specs/pull/5491#discussion_r270517299) [](ancestors = 270517299)